upi/vsphere/modules/rhcos_virtual_machines/main.tf

```diff
@@ -28,74 +28,58 @@ module "bootstrap" {
   source = "./machine"
 
   name             = "bootstrap"
-  instance_count   = "${var.bootstrap_complete ? 0 : 1}"
-  ignition_url     = "${var.bootstrap_ignition_url}"
+  instance_count   = "${var.bootstrap_count}"
+  ignition         = "${file("${var.bootstrap_ignition_path}")}"
   resource_pool_id = "${module.resource_pool.pool_id}"
   datastore        = "${var.vsphere_datastore}"
   folder           = "${module.folder.path}"
   network          = "${var.vm_network}"
   datacenter_id    = "${data.vsphere_datacenter.dc.id}"
   template         = "${var.vm_template}"
   cluster_domain   = "${var.cluster_domain}"
-  ipam             = "${var.ipam}"
-  ipam_token       = "${var.ipam_token}"
-  ip_addresses     = ["${compact(list(var.bootstrap_ip))}"]
+  ip_addresses     = ["${compact(list(var.bootstrap_ip_address))}"]
   machine_cidr     = "${var.machine_cidr}"
   memory           = "8192"
   num_cpu          = "4"
+  dns_addresses    = "${var.vm_dns_addresses}"
 }
 
-module "control_plane" {
+module "master" {
```

Comment:
I know the AWS Terraform uses master/worker, but if you already have control-plane/compute in the vSphere Terraform I'd rather stay with it instead of backsliding.  See also #1330.